agent/bin/kontena-agent

```diff
@@ -9,19 +9,25 @@ Docker.validate_version!
 $stdout.sync = true
 $0 = 'kontena-agent'
 
-api_token = ENV['KONTENA_TOKEN']
 api_uri = ENV['KONTENA_URI']
+grid_token = ENV['KONTENA_TOKEN']
+node_token = ENV['KONTENA_NODE_TOKEN']
 
-if api_token.nil? || api_token.empty?
-  abort('Configuration error: The KONTENA_TOKEN env is required')
-end
 if api_uri.nil? || api_uri.empty?
   abort('Configuration error: The KONTENA_URI env is required')
 end
 if api_uri.match(/^http.*/)
   api_uri = api_uri.sub('http', 'ws')
 end
 
+if grid_token && !grid_token.empty?
+
+elsif node_token && !node_token.empty?
+
```

Comment:
Looks a bit weird having empty if/elsif blocks. I get the point, but maybe some other styling would read better?